restaurant/src/app/admin/foodsold/foodsold.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+import { ViewSoldFoodsComponent } from '../view-sold-foods/view-sold-foods.component';
+import { viewFood } from '../viewFood.model';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+interface FoodSold {
```

Comment:
instead of creating an interface and DTO class inside a component, create a separate folder for DTO and have DTOs there